static/app/views/alerts/rules/details/body.tsx

```diff
@@ -232,16 +241,34 @@ export default class DetailsBody extends React.Component<Props> {
       : null;
 
     return (
-      <StatusContainer>
-        <HeaderItem>
-          <Heading noMargin>{t('Current Status')}</Heading>
-          <Status>
-            <AlertBadge status={status} hideText />
-            {activeIncident ? t('Triggered') : t('Resolved')}
-            {activityDate ? <TimeSince date={activityDate} /> : '-'}
-          </Status>
-        </HeaderItem>
-      </StatusContainer>
+      <Projects orgId={orgId} slugs={projectSlugs}>
+        {({initiallyLoaded, projects}) => {
+          return initiallyLoaded ? (
```

Comment:
something seems weird about this i'm not sure how we normally do it

Reply:
yeah, it seems pretty gross to me too